bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java

```diff
@@ -338,24 +343,26 @@ private Build getNpmBuild(
 
     private void addLogs(BuildRecord buildRecord, Set<Logfile> logs) {
         String externalBuildID = buildRecord.getId().toString();
-        if (buildRecord.getBuildLogSize() != null && buildRecord.getBuildLogSize() > 0) {
+        long buildLogSize = bifrost.getFinalLogSize(buildRecord.getId().toString(), "build-log");
```

Comment:
the processContext for build logs should be `build-<<buildId>>` right?

Reply:
I belive it's only buildId, right @thescouser89 ?

Reply:
I have a feeling it's just <<buildId>>. But I'm using the IdConverter to switch between process context and the "long id" and it's been working fine.